CHANGELOG.md

```diff
@@ -1,24 +1,9 @@
+# 0.10.11-dev
+
 ## 0.10.10
 
 * Stable release for null safety.
 
-## 0.10.10-nullsafety.3
```

Comment:
@natebosch are we generally removing these once stable is released? Given these are real versions on pub I am not sure why we would remove them?

Reply:
They are generally hidden from users. At this point, a user would have to explicitly opt-in to these. I feel they are better hidden.

Reply:
Yeah, we started collapsing these logs down to just the stable ones very late into the process when we were publishing for stable null safety. For the packages where we hadn't done originally that we've been going back and collapsing it. It's useful in the github history, but I don't think it's worth the noise to keep it in the file.